manim/mobject/probability.py

```diff
@@ -27,6 +27,24 @@
 
 
 class SampleSpace(Rectangle):
+    """A sample space.
+
+    Examples
+    --------
+    .. manim:: ExampleSampleSpace
+        :save_last_frame:
+
+        class ExampleSampleSpace(Scene):
+            def construct(self):
+                poly1 = SampleSpace(stroke_width=15,fill_opacity=1)
+                poly2 = SampleSpace(width = 5, height = 3, stroke_width=5,fill_opacity=0.5)
+                poly3 = SampleSpace(width = 2, height = 2, stroke_width=5,fill_opacity=0.1)
```

Comment:
Again, you should use the bot's `!mdocstring` command.

Reply:
Copied the documentation from the bot. @naveen521kk 
Hopefully it should be fine now.